processor/logcountprocessor/record.go

```diff
@@ -55,9 +58,10 @@ func convertToRecords(logs plog.Logs) []Record {
 // convertToRecord converts a log record to a simplified representation.
 func convertToRecord(log plog.LogRecord, resource map[string]any) Record {
 	return Record{
-		resourceField:   resource,
-		attributesField: log.Attributes().AsRaw(),
-		bodyField:       log.Body().AsRaw(),
-		severityField:   log.SeverityText(),
+		resourceField:       resource,
+		attributesField:     log.Attributes().AsRaw(),
+		bodyField:           log.Body().AsRaw(),
+		severityTextField:   log.SeverityText(),
+		severityNumberField: log.SeverityNumber().String(),
```

Comment:
Just one last comment here; Should this be the actual number instead of the string, so that comparison operators can be used against it?

Reply:
Let me see if it's possible. I originally tried pulling that out as a number, but I didn't see any easy functions for providing that, so I defaulted to the string.

Reply:
@jmwilliams89 You should be able to just cast it `int32(log.SeverityNumber())` (if I understand the docs right: https://pkg.go.dev/go.opentelemetry.io/collector/pdata/plog#SeverityNumber)

Reply:
Looks like it gets returned as an int32, so I think we can use that. Do you think the severity text or the string representation of severity number are more useful for the user?